apps/src/aiTutor/types.ts

```diff
@@ -68,9 +68,10 @@ export interface Level {
 
 export interface ChatContext {
   // studentInput is the last user message for general chat
-  // or the student's code for compilation and validaiton.
+  // or the student's code for compilation and validation.
   studentInput: string;
-  tutorType: AITutorTypesValue | undefined;
+  studentCode: string;
```

Comment:
Added the student code to the `ChatContext` because I'll now need to include it in the formattedQuestion along with the `studentInput`, since I'm not using three different system prompts anymore.